docs/asciidoc/static/breaking-changes.asciidoc

```diff
@@ -21,11 +21,11 @@ Be sure to specify the correct value for the `--version` option during installat
 [float]
 ==== Configuration Changes
 
-The Elasticsearch output plugin configuration has the following changes:
+The [Elasticsearch output plugin](https://www.elastic.co/guide/en/logstash/2.0/plugins-outputs-elasticsearch.html) configuration has the following changes:
 
 * The `host` configuration option is now `hosts`, allowing you to specify multiple hosts and associated ports in the 
 `myhost:9200` format
-* New options: `bind_host`, `bind_port`, `cluster`, `embedded`, `embedded_http_port`, `port`, `sniffing_delay`
+* New options: `sniffing_delay`
```

Comment:
This is a good step, but is not sufficient.  Those options _do_ exist, and they should be mentioned somewhere in or after line 31, as these options (minus `port` and `embedded`) do still exist for the `elasticsearch_java` output plugin.